bundles/net.tourbook.device.garmin.fit/src/net/tourbook/device/garmin/fit/listeners/MesgListener_Event.java

```diff
@@ -108,13 +108,9 @@ public void onMesg(final EventMesg mesg) {
          // Garmin: Elapsed, Timer, and Moving Durations
          // https://developer.garmin.com/fit/cookbook/durations/
 
-         case START:
+         case START -> handleTimerStartEvent(javaTime);
```

Comment:
@wolfgang-ch  Using the [Java 14 Switch expression](https://blog.gypsyengineer.com/en/tech/new-switch-expressions-in-java-14.html) to ensure that Java 17 is actually used